src/ui/public/courier/search_source/search_source.js

```diff
@@ -601,7 +602,11 @@ export function SearchSourceProvider(Promise, Private, config) {
             _.set(flatData.body, '_source.includes', remainingFields);
           }
 
-          flatData.body.query = buildESQuery(flatData.index, flatData.query, flatData.filters);
+          try {
+            flatData.body.query = buildESQuery(flatData.index, flatData.query, flatData.filters);
+          } catch (e) {
+            throw new KbnError(e.message, KbnError);
```

Comment:
Ok now that I see this, I'm assuming that's why you removed the wildcard error type?